Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/Security - Personal Microsoft Accounts Blocked.json

```diff
@@ -8,6 +8,6 @@
       }
     }
   ],
-  "description": "Disables the addition of personal Microsoft accounts to the device to ensure DLP.",
+  "description": "What is it: Disables the addition of personal Microsoft accounts to the device to ensure DLP. Learn More: https://docs.microsoft.com/en-us/mem/intune/configuration/device-restrictions-windows-10#cloud-and-storage",
```

Comment:
**What does this do?**
Disables the addition of personal Microsoft accounts to the device to ensure Data Loss Prevention. 

**Why should you use this?**
If you want to prevent users from adding personal Microsoft accounts to Windows, which would allow them to transfer data outside of your organization. 

**What is the end-user impact?**
Users cannot connect Windows to personal Microsoft accounts.

**Learn more**
[Microsoft accounts configuration](https://docs.microsoft.com/en-us/mem/intune/configuration/device-restrictions-windows-10#cloud-and-storage)